templates.ts

```diff
@@ -64,6 +64,9 @@ export function generateRendererFile(viewHash: string, viewFunction: string, dec
     } else {
         initBody = initBodyWithoutDecoder(viewHash, viewFunction);
     }
+    let newLinesStr = newLines !== undefined ? (newLines.toString().charAt(0).toUpperCase() + newLines.toString().slice(1)) : "True";
+    let indentStr = indent !== undefined ? indent.toString() : "4";
+    let optionsSet = "options = { defaultFormatOptions | newLines = " + newLinesStr +", indent = " + indentStr + " }";

```

Comment:
Use 
```
`options { newLines = ${newLines}, indent = ${indent}`
```

instead

Reply:
you mean that is what optionsSet should be assigned?

Reply:
I mean use template literals: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals